### DISCUSSION TITLE
Structs e Método Construtor

### DISCUSSION THREAD

**Discussion by ghost** [balta-io/2801#79] [Perguntas e Respostas] [answered]
Não precisamos especificar o tipo de retorno de um método construtor, mas, por debaixo dos panos, o tipo de retorno dele é o definido pela struct? (Nesse caso, não precisar escrever o tipo e o return é um sugar syntax?)

**Comment by andrebaltieri** [comment, marked as answer]
Exatamente @PolarisTDM, não precisamos definir o tipo de retorno para métodos construtores... será sempre a própria classe ou struct.
